docs/reference-architectures/app-service-web-app/baseline-zone-redundant-content.md

```diff
@@ -325,11 +325,11 @@ App Service has built-in and integrated monitoring tools that you should enable
 
 ## Governance
 
-Web apps benefit from Azure Policy by enforcing architectural and security decisions. Azure Policy can make it (1) impossible to deploy (deny) or (2) easy to detect (audit) configuration drift from your preferred desired state. This helps you catch IaC deployments or Azure portal changes that deviate from agreed upon architecture. You should place all resources in your architecture under Azure Policy governance. Use built-in policies or policy initiatives where possible to enforce key network topology, service feature, security, and monitoring decisions, for example:
+Web apps benefit from Azure Policy by enforcing architectural and security decisions. Azure Policy can make it (1) impossible to deploy (deny) or (2) easy to detect (audit) configuration drift from your preferred desired state. This helps you catch IaC (Infrastructure as Code) deployments or Azure portal changes that deviate from agreed-upon architecture. You should place all resources in your architecture under Azure Policy governance. Use built-in policies or policy initiatives where possible to enforce essential network topology, service features, security, and monitoring decisions, for example:
 
 - App Service should disable public network access
 - App service should use virtual network integration
-- App Service should use private link
+- App Service should use a private link
```

Comment:
I don't think a works for Private Link. However, I think it should be capitalized. How about:
App Service should use Azure Private Link to connect to PaaS services

Reply:
Updated.